### ISSUE TITLE
`Plog.e: Missing field: institution_id` on oauth 

### ISSUE THREAD

**Issue by ir2pid** [plaid/plaid-link-android#150]
# The problem
I get a log errors when using oauth in sandbox mode. Regular transaction works okay, only oauth gives an error from sdk with error: `Plog.e: Missing field: institution_id` and `Plog.e: Missing field: institution_name`

# Environment

|                        |  |
| ------------------------| ---------- |
| Android OS Version                    |  11                |
| Android Devices/Emulators    | Samsung galaxy s20+                |

# Steps to Reproduce

in sandbox mode, 
use platypus oauth, 
click grant access, 
control returns to app and logcat throws the below error.

# Expected Result
no errors in log 

# Logs
```
2021-03-30 09:56:53.167 10385-10385/com.xx.yy I/cr0: Url to be loaded https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.5&androidSdkImplementation=inProcess&token=link-sandbox-xxyyzz
2021-03-30 09:56:53.174 10385-11253/com.xx.yy W/a.moneytransfe: Accessing hidden method Landroid/media/AudioManager;->getOutputLatency(I)I (greylist, reflection, allowed)
2021-03-30 09:56:53.185 10385-11253/com.xx.yy W/cr_media: Requires BLUETOOTH permission
2021-03-30 09:56:54.582 10385-10385/com.xx.yy I/chromium: [INFO:CONSOLE(0)] "Access to link element resource at 'https://cdn.plaid.com/experiments/optimizely/datafile' from origin 'https://secure.plaid.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.", source: https://secure.plaid.com/link/v2/stable/link.html?isWebview=true&isMobile=true&apiVersion=v2&linkSdkVersion=3.2.5&androidSdkImplementation=inProcess&token=link-sandbox-xxyyzz (0)
2021-03-30 09:56:56.221 10385-11177/com.xx.yy V/FA: Inactivity, disconnecting from the service
2021-03-30 09:56:57.377 10385-10385/com.xx.yy I/Event: LinkEvent(eventName=acknowledged, metadata=LinkEventMetadata(errorCode=null, errorMessage=null, errorType=null, exitStatus=null, institutionId=null, institutionName=null, institutionSearchQuery=null, linkSessionId=, mfaType=null, requestId=, timestamp=, viewName=null, metadataJson=null))
2021-03-30 09:56:57.418 10385-10385/com.xx.yy D/ax0: Event name: OPEN
2021-03-30 09:56:57.448 10385-10385/com.xx.yy E/a: Plog.e: Missing field: institution_id
    java.lang.Throwable
        at com.plaid.internal.p$a.a(SourceFile:624)
        at com.plaid.internal.a.a(SourceFile:17)
        at com.plaid.link.event.LinkEvent$Companion.fromMap$link_sdk_base_release(SourceFile:10)
        at com.plaid.internal.ax0.a(SourceFile:59)
        at com.plaid.internal.vq0.a(SourceFile:1)
        at com.plaid.internal.jx0.shouldOverrideUrlLoading(SourceFile:2)
        at org.chromium.android_webview.AwContentsClientBridge.shouldOverrideUrlLoading(chromium-TrichromeWebViewGoogle6432.aab-stable-438910534:14)
        at android.os.MessageQueue.nativePollOnce(Native Method)
        at android.os.MessageQueue.next(MessageQueue.java:335)
        at android.os.Looper.loop(Looper.java:206)
        at android.app.ActivityThread.main(ActivityThread.java:8419)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:596)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
2021-03-30 09:56:57.459 10385-10385/com.xx.yy E/a: Plog.e: Missing field: institution_name
    java.lang.Throwable
        at com.plaid.internal.p$a.a(SourceFile:624)
        at com.plaid.internal.a.a(SourceFile:17)
        at com.plaid.link.event.LinkEvent$Companion.fromMap$link_sdk_base_release(SourceFile:11)
        at com.plaid.internal.ax0.a(SourceFile:59)
        at com.plaid.internal.vq0.a(SourceFile:1)
        at com.plaid.internal.jx0.shouldOverrideUrlLoading(SourceFile:2)
        at org.chromium.android_webview.AwContentsClientBridge.shouldOverrideUrlLoading(chromium-TrichromeWebViewGoogle6432.aab-stable-438910534:14)
        at android.os.MessageQueue.nativePollOnce(Native Method)
        at android.os.MessageQueue.next(MessageQueue.java:335)
        at android.os.Looper.loop(Looper.java:206)
        at android.app.ActivityThread.main(ActivityThread.java:8419)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:596)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
2021-03-30 09:56:57.461 10385-10385/com.xx.yy I/Event: LinkEvent(eventName=OPEN, metadata=LinkEventMetadata(errorCode=, errorMessage=, errorType=, exitStatus=, institutionId=, institutionName=, institutionSearchQuery=, linkSessionId=4bc31efc-9b52-40c0-b4a4-5f9dd3bd68e0, mfaType=, requestId=, timestamp=2021-03-30T07:56:57.414Z, viewName=com.plaid.link.event.LinkEventViewName$CONSENT@f0ee927, metadataJson=null))

```

**Comment by amytang0** [comment]
Hi, which SDK version are you using?

**Comment by ir2pid** [comment]
this `com.plaid.link:sdk-core:3.2.5`

**Comment by ir2pid** [comment]
> Hi, which SDK version are you using?

this com.plaid.link:sdk-core:3.2.5

**Comment by amytang0** [comment]
Thanks for the flag! We will remove this log in our next release, apologies

**Comment by JeroenMols** [comment]
@ir2pid Are you still seeing this issue on SDK version `3.3.0` or higher?
Thank you for checking, that allows us to close this issue.

**Comment by JeroenMols** [comment]
We haven't heard back from you in a while, so we're assuming this issue is fixed.
Should that not be the case, feel free to reopen or file a new ticket!
